vm-systemd/qubes-relabel-root.service

```diff
@@ -0,0 +1,20 @@
+[Unit]
+Description=Relabel /
```

Comment:
How this service relates to the standard relabel service? Wouldn't it duplicate most work (taking twice as long)? If the standard relabel is now redundant, maybe it can be disabled (or replaced by this one)?

Reply:
The standard service is quite a bit smarter in various ways, but it is buggy in this one.  The best option I can think of is to patch the standard service.

Reply:
Looks like the standard service does a few different things:

- Futz with the UEFI boot order to account for dual-boot machines
- Check `/etc/selinux/config` for SELinux being in minimal, targeted, or MLS (multi-level security) mode
- Handle all filesystems from `/etc/fstab`
- Reboot afterwards

Of these, only the last makes sense in Qubes OS, and only if PID 1 was mislabeled.  Thoughts?